src/frontend/components/PlaylistPortability/index.spec.tsx

```diff
@@ -0,0 +1,191 @@
+import { act, fireEvent, render, screen } from '@testing-library/react';
+import fetchMock from 'fetch-mock';
+import React from 'react';
+
+import { PlaylistPortability } from '.';
+import {
+  playlistLiteMockFactory,
+  playlistMockFactory,
+  videoMockFactory,
+} from '../../utils/tests/factories';
+import { wrapInIntlProvider } from '../../utils/tests/intl';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { Deferred } from '../../utils/tests/Deferred';
+import { wrapInRouter } from '../../utils/tests/router';
+import { Grommet } from 'grommet';
+import MatchMediaMock from 'jest-matchmedia-mock';
+import MatchMedia from 'jest-matchmedia-mock';
```

Comment:
You are importing twice the same thing. A module can only have one `default` export.